src/services/profile.ts

```diff
@@ -4,7 +4,7 @@ import { S3Client } from '@aws-sdk/client-s3';
 import moment from 'moment';
```

Comment:
Are we adding this profile service as any route ?

Reply:
If not, shall we add a router for this ?

Reply:
This is a non-blocking comment

Reply:
it is part of new routes @sanpj2292

Reply:
Oh... nice 👍🏼